libcontainer/cgroups/cgroups.go

```diff
@@ -46,6 +46,9 @@ type Manager interface {
 
 	// Sets the cgroup as configured.
 	Set(container *configs.Config) error
+
+	// Gets the cgroup as configured.
+	GetCgroups() (*configs.Cgroup, error)
```

Comment:
Hey @devimc:

1. nit: s/splications/applications on your commit message.
2. Can you better explain the intended use case, highlighting why this is needed? Specific to the Kata case (in both v2-shim and CLI based configurations), what do we do with the cgroup config?  Do we need to store a copy of the information, or can it be obtained as needed later? It just isn't clear to me. thx!

Reply:
done, thanks